static/app/components/searchQueryBuilder/valueCombobox.tsx

```diff
@@ -776,16 +778,11 @@ export function SearchQueryBuilderValueCombobox({
       }
 
       // If at the start of the input and backspace is pressed, delete the last selected value
```

Comment:
I don't think this comment is accurate anymore

Reply:
You are correct!